Makefile

```diff
@@ -2,7 +2,7 @@
 #
 # It's auto-generated by sonata-project/dev-kit package.
```

Comment:
The `Makefile` is for dev and test of the bundle. This is not for production integration.

Why not just add this on the doc?

In a more global way, we should decide the way to go to manage assets on **each** bundle.

See https://github.com/sonata-project/dev-kit/issues/137 for example.